deploy/policy/templates/project-authn-def.template.sh.yml

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
how about changing the file name to `conjur-authenticator` or `conjur-authn-k8s` or `conjur-authn-k8s-authenticator` (you get the idea)